benchmarks/microbenchmarks/test/benchmark_config.yml

```diff
@@ -0,0 +1,19 @@
+# Sample configuration for inference kernel benchmarks
+quantization_config_recipe_names:
+  - "baseline"
+  - "int8wo"
+  - "int4wo-128"
+output_dir: "benchmarks/microbenchmarks/test/results"  # Directory for results and plots
+model_params:
+  matrix_shapes:
+    - name: "custom"
+      shapes: [
+        [1024, 1024, 1024],  # [m, k, n]
+        [2048, 4096, 1024],
+        [4096, 4096, 1024]
+      ]
+  high_precision_dtype: "torch.bfloat16"
+  compile: true
```

Comment:
nit: I tend to prefer to consolidate multiple variables into 1, like compile vs compile_mode is kind of redundant when you could just have 

compile: "max-autotune" or compile: "false"